frontend/src/utils/YupSchema.ts

```diff
@@ -212,7 +212,7 @@ export const ParcelSchema = Yup.object()
           .required('PID or PIN Required')
           .max(9, 'Please enter a valid PIN no longer than 9 digits.'),
       }),
-      buildings: Yup.array().of(Building),
+      buildings: Yup.array(),
```

Comment:
No point in validating the associated buildings as they can't be corrected in this workflow anyways.